WooCommerce/Classes/ViewModels/Order Details/OrderDetailsPaymentAlerts.swift

```diff
@@ -48,6 +48,17 @@ final class OrderDetailsPaymentAlerts {
         modalController?.setViewModel(viewModel)
     }
 
+    func nonRetryableError(from: UIViewController?, error: Error) {
+        let viewModel = nonRetryableErrorViewModel(amount: amount, error: error)
+        let newAlert = CardPresentPaymentsModalViewController(viewModel: viewModel)
+        modalController?.setViewModel(viewModel)
```

Comment:
The next line blows this away kinda. Maybe `guard` on `nil` `modalController` and then call this then `return`? Similar to how `setViewModelAndPresent` works in `CardReaderSettingsAlerts`?